src/index.js

```diff
@@ -13,7 +13,9 @@ ReactDOM.render(
     <BrowserRouter>
       <AuthenticationProvider>
```

Comment:
suggestion: like you do for components, you can export all of them from an index